.github/workflows/darwin.yaml

```diff
@@ -0,0 +1,20 @@
+name: Darwin
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+jobs:
+  build:
+    name: Unit Tests
+    runs-on: macos-latest
+    steps:
+    - name: Set up Go 1.x
+      uses: actions/setup-go@v2
+      with:
+        go-version: ^1.13
```

Comment:
Can we use go1.15?

Reply:
pls follow this up later @mayankshah1607 thanks.